pom.xml

```diff
@@ -56,13 +56,16 @@
   </issueManagement>
 
   <properties>
-    <java.version>1.8</java.version>
+    <java.version>11</java.version>
```

Comment:
`neo4j-harness` is used for tests only. I don't think we should upgrade java version from 8 to 11.
Java 8 is supported by driver and most widely used.
That's very strange that harness supports only java 11+.
Is it in release notes somewhere?

Reply:
Yes, it would be nice the work around this. I couldn't find any release notes for `neo4-harness` at all.
When using java 1.8 I get:
```
Error:line (30)java: cannot access org.neo4j.harness.junit.rule.Neo4jRule
  bad class file: /.m2/repository/org/neo4j/test/neo4j-harness/4.0.0/neo4j-harness-4.0.0.jar!/org/neo4j/harness/junit/rule/Neo4jRule.class
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

Reply:
Well, JDK 11 is a minimal supported version for 4.0
https://neo4j.com/docs/operations-manual/4.0-preview/installation/requirements/#deployment-requirements-software